app/scripts/src/config.js

```diff
@@ -28,5 +32,12 @@ module.provider('languageSelectConfig', function () {
         setDefaultLanguage: function (value) {
             defaultLanguageId = value;
         },
+        setReloadOnChange: function (value) {
+            if (value === false || value === null || value === 0) {
+                reloadOnChange = false;
+            } else if (typeof value !== 'undefined') {
+                throw new Error('Cannot set reloadOnChange to ' + JSON.stringify(value) + ' [' + typeof value + ']');
+            }
```

Comment:
I don't think it's useful to allow empty calls on this function, it would be easier to do a type check:

```js
if (typeof value !== 'boolean') {
    throw new Error('Value should be a boolean')
}
reloadOnChange  = value;
```